quarkus/admin/src/main/resources/org/apache/polaris/admintool/banner.txt

```diff
@@ -0,0 +1,15 @@
+           _                                              ___
+          |_) _  |  _. ._ *  _     /\   _| ._ _  * ._      |  _   _  |
+          |  (_) | (_| |  | _>    /--\ (_| | | | | | |     | (_) (_) |
```

Comment:
I find this pretty hard to read, maybe it's just github?

The existing banner seems OK to me